pdc/apps/package/models.py

```diff
@@ -27,6 +27,7 @@ class RPM(models.Model):
     release             = models.CharField(max_length=200, db_index=True)
     arch                = models.CharField(max_length=200, db_index=True)  # nosrc
     srpm_name           = models.CharField(max_length=200, db_index=True)  # package (name of srpm)
+    koji_build_target   = models.CharField(max_length=200, null=True, blank=True, db_index=True)
```

Comment:
RPM has linked_releases  
 linked_releases     = models.ManyToManyField('release.Release', related_name='linked_rpms')
Is it what you want?